src/api/spec/lib/feature_switch/feature_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Feature do
+  let(:user) { build(:confirmed_user, login: 'Tom') }
+  subject { Feature.active?('bootstrap') }
+
+  context 'with beta users' do
+    before do
+      allow(User).to receive(:possibly_nobody).and_return(user)
```

Comment:
Why don't you use the login helper?

Reply:
This helper is not accessible  under the spec/lib directory. So I decided to use the next aproach:
```
User.session = user
```